semeio/jobs/overburden_timeshift/ots.py

```diff
@@ -14,6 +14,7 @@
 from semeio.jobs.overburden_timeshift.ots_vel_surface import OTSVelSurface
 from semeio.jobs.overburden_timeshift.ots_res_surface import OTSResSurface
 from semeio.jobs.overburden_timeshift.ots_config import build_schema
+from semeio._exceptions.exceptions import ConfigurationError
```

Comment:
The other jobs should use it too I reckon

Reply:
If you agree that this is the right approach, then yes I agree, we could do this for the rest of the scripts as well.